### DISCUSSION TITLE
Blazor: Meu css scoped não aplica no botão apagar

### DISCUSSION THREAD

**Discussion by oimarcella** [balta-io/forum#664] [Tira Dúvidas] [answered]
Nessa imagem eu peguei na aba network do webtools, e vi que o css está sendo carregado, porém não aplica de jeito nenhum.
![image](https://github.com/user-attachments/assets/64eedbdb-bacf-49ee-bc73-dd249427faa7)

Minha estrutura:
![image](https://github.com/user-attachments/assets/69eace40-3cd8-4319-ad03-18a1c5ab94e0)

![image](https://github.com/user-attachments/assets/dbe7f542-423e-45fd-88cd-3949447cdb7d)

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @oimarcella, como vai?

O CSS parece estar sendo aplicado a página... o que exatamente você está querendo fazer e que o CSS não foi aplicado?

**Reply by oimarcella** [reply]
No botão apagar eu apliquei cores, font-weight, uppercase na letra, background e nada disso foi aplicado.
O css está ali no print também.

Eu sei que o css não está com código incorreto pois, eu estava aplicando ele inline e funcionava, porém quando mudei pro arquivo css ele não funciona.

**Reply by andrebaltieri** [reply]
Certo, mas você aplicou em qual elemento? Como nomeou esta classe? Como referenciou esta classe no elemento?

**Reply by oimarcella** [reply]
Está com as classes .btn e .btn-delete, estou aplicando nessas classes o CSS
```
<li style="padding: 11px 14px;cursor:pointer;border: 1px solid lightgray;border-left: 0;border-right: 0;">
            <a href="/products/edit/@produto.Id">@produto.Title</a>
            <a class="ml-3 btn btn-delete" href="/products/delete/@produto.Id">Apagar</a>
        </li>
```

```
.btn .btn-delete {
    background: red;
    margin-left: 10px;
    color: #fff;
    padding: 6px 8px;
    cursor: pointer;
    border-radius: 6px;
    font-size: 11px;
    text-decoration: none;
    text-transform: uppercase;
    font-weight: 600;
}
```

**Reply by oimarcella** [reply]
A escrever acabei de reparar que no CSS as duas classes deveriam ser escritas juntas .btn.btn-delete.

KKKK que engano meu!
